### ISSUE TITLE
Declare Browser Compatibility

### ISSUE THREAD

**Issue by aspidites** [qmlweb/qmlweb#38]
At the moment, I don't know which browsers this project is meant to work on. It would be nice to know, for instance, if this relies on technology which precludes running a qmlweb app on IE8. 

Declaring a set of supported browsers would also allow choices to be made about supporting libraries and standards. For instance, if only IE10+ is required, I'd then wonder if using ES2015 or TypeScript would make development more sane than plain ol' JavaScript. 

Obviously sticking to plain JavaScript has the benefit of being ubiquitously understood, though. 

If legacy browsers aren't a concern, it might be nice to state that we officially support the latest x stable releases of Firefox, Chrome, and IE (and whoever else is a big player nowadays). More accurate might be to declare supported version of rendering engines, which would prevent us from alienating niche browsers. dd

**Comment by ChALkeR** [comment]
@aspidites This would probably require merging to be done first.
After merging, I plan to do some cleanup, then use `babel` in the build process to give us an opportunity to use newer language version. Also, we could add some polyfills for not so recent browsers. After that is done, we could take a look at what we got and try to declare the list of supported browsers and their versions =).

**Comment by aspidites** [comment]
Sounds like a plan! From professional experience, I'd really hope we weren't worried about IE 8.

**Comment by ChALkeR** [comment]
I don't think that we are worried about IE 8 in a sense that we won't monkey-patch global objects with polyfills (seel f33d663710690a78900b014ea2f437c56a31bedf, for example), users could do that on their own.

But we will use babel to transplit everything to ES5.

**Comment by aspidites** [comment]
:+1:

**Comment by ChALkeR** [comment]
The lower bar should probably be somewhere around IE11+/SF9+ for now.
IE10 is dead long ago, judging by the usage stats.

Perhaps it would make sense to drop IE11/SF9 as the next step in some time.

/cc @qmlweb/collaborators, thoughs?

**Comment by ChALkeR** [comment]
To clarify: I mean minimal support here, when everything is running and not crashing. Some specific functionality might be degraded or unavailable on older browsers.
